### PR TITLE
Feature/refresh groups when entering to groups page

### PR DESCRIPTION
**Pull request by titandiaz**

### CODE REVIEW

File: components/http-list/OkHttpList.vue [hunk 1]
```diff
@@ -206,6 +205,9 @@
         created() {
             this.listKey = `l-${this.utilsService.generateUuid()}`;
         }
+        async activated(){
```
Comment (EdisonArango) [inline components/http-list/OkHttpList.vue:208]:
```suggestion

        async activated(){
```

File: pages/home/pages/community/components/shared/OkCommunityProfileActionButtons.vue [hunk 2]
```diff
@@ -74,7 +73,7 @@
             this.canBanOrUnban = loggedInUser.canBanOrUnbanUsersInCommunity(this.community);
         }
 
-        async onButtonClicked() {
+        async OpenPostModal() {
```
Comment (EdisonArango) [inline pages/home/pages/community/components/shared/OkCommunityProfileActionButtons.vue:76]:
```suggestion
        async openPostModal() {
```

File: components/http-list/OkHttpList.vue [hunk 3]
```diff
@@ -189,7 +188,7 @@
 
         items: T[] = [];
 
-        listKey: String;
+        listKey: String = '';
```
Comment (EdisonArango) [inline components/http-list/OkHttpList.vue:191]: Why is this change necessary?

Comment (titandiaz) [reply components/http-list/OkHttpList.vue:191]: because the variable is not initialized if it does not have a defined value, it does not exist
